### ISSUE TITLE
Linking template wizard.html and step.html

### ISSUE THREAD

**Issue by eatshootrock** [mgonto/angular-wizard#205]
I'm quite new with AngularJS and angular-wizard. I was able to do a simple wizard and found out that you can use external template file (ex. wizard.html) to further customize the navigation by using

`<wizard on-finish="finishedWizard()" indicators-position="top" template="app/views/wizard.html">`

However, I am also interested in linking another template called "step.html" to further customize my html.  

basically I would like to use an external wizard.html and step.html for my project.

Is there a way to do this without editing "angular-wizard.js"'s code?

**Comment by jacobscarter** [comment]
Not currently, this would be considered a feature request
